index.html

```diff
@@ -199,11 +207,13 @@ <h3>
             <a href="./index.html" class="get buttons">
                 Get my resume
             </a>
-        </div>
+        </div> 
         <ul class="langInfo">
-            <h3 langs>Languages
+            
+            <h3 class = "langs" id="langs">Languages
                 <p class="v">v</p>
             </h3>
+            <div class="plangsContainer">
             <li class="pLangs javaScript">
                 <img class="image" src="./images/Ellipse 1 (1).png"></img>
                 <h2>JavaScript</h2>
```

Comment:
- [ ] While you did a wonderful job styling your project :+1: , as a project requirement you should make the language, skills, and framework sections to be dropdown menus and add 5 tags per row listing the technologies :+1:. You can check this [video](https://www.youtube.com/watch?v=eKo1NV1qxbc) where it is done using only HTML and CSS.